docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -52,7 +52,7 @@ I have:
      - [ ] If I am fixing just part of the issue, I have just referenced it w/o any of the "fixes” keywords.
 - [ ] Created the issue I am fixing/addressing, if it was not present.
 - [ ] Formatted all JavaScript and TypeScript code that was changed. (use the [prettier extension](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) or run `npm run prettier-format`)
-- [ ] Confirmed that there are no linter warnings or errors (use the [eslint extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) or run `npm run eslint`).
+- [ ] Confirmed that there are no linter warnings or errors (use the [eslint extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint), run `npm run eslint` before creating your PR, or run `npm run eslint-watch` to eslint as you go).
```

Comment:
@Cyrik does the rest of the pipeline still run when the formatting test fails? If not, then at least that should be fixed for the non-release workflow, I think.

Reply:
Yeah, the rest does run, which is pretty nice. 
It's mostly a small annoyance to add a second PR with the format/lint fix if you forget to run the tool, which is why I like hooks. But I also see how hooks can be annoying because they take time to run... So I'm also ok not adding them.

Reply:
I'm running with format on save now.

Reply:
Ah, that's smart. Now I just need the same thing for the new es-lint thing.
@corasaurus-hex Is there something that shows me es-lint problems inline, like clj-kondo -> clojure-lsp -> Calva, does in my Clojure code?

Reply:
I use the eslint extension for vs code which will give me all those underlines inline https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint

Reply:
Ah nice, I had that already. Thank you.